ui/raidboss/data/04-sb/dungeon/temple_of_the_fist.js

```diff
@@ -51,16 +61,21 @@
     },
     {
       id: 'Temple Fourfold Shear',
-      regex: / 14:1FD9:Arbuda starts using Fourfold Shear on (\y{Name})/,
+      regex: Regexes.startsUsing({ id: '1FD9', source: 'Arbuda' }),
+      regexDe: Regexes.startsUsing({ id: '1FD9', source: 'Arbuda' }),
+      regexFr: Regexes.startsUsing({ id: '1FD9', source: 'Arbuda' }),
+      regexJa: Regexes.startsUsing({ id: '1FD9', source: 'アブダ' }),
+      regexCn: Regexes.startsUsing({ id: '1FD9', source: '额部陀' }),
+      regexKo: Regexes.startsUsing({ id: '1FD9', source: '아부다' }),
       infoText: function(data, matches) {
-        if (data.me == matches[1]) {
+        if (data.me == matches.target) {
           return {
             en: 'Tank buster on YOU',
           };
         }
         if (data.role == 'healer') {
           return {
-            en: 'Tank buster on ' +data.shortName(matches[1]),
+            en: 'Tank buster on ' +data.shortName(matches.target),
```

Comment:
Can you add a space around this plus sign while you're here? I'm surprised there's no lint rule for that.